### ISSUE TITLE
i want to change password

### ISSUE THREAD

**Issue by dlacofla** [RocketChat/Rocket.Chat#27451]
hello,
i lost my Password... 
i tried to reset my password in rocket chat and rocket chat sent me the email but, I didn't get an email
how can I change my password?
thanks.

**Comment by MarlynxLinux** [comment]
yes good question, i also wanted to reset my password on the server open.rocket.chat, I don't receive a password reset link in my email on gmail

**Comment by WeOpenChat** [comment]
Yes
The problem with reset password.
The email was sent without link.

**Comment by shubham21155102** [comment]
Assign this issue to me
